src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1746,6 +1748,8 @@ type RentalVehicle implements Node & PlaceInterface {
   rentalUris: VehicleRentalUris
   "ID of the vehicle in the format of network:id"
   vehicleId: String
+  "The vehicle rental system information."
+  vehicleRentalSystem: VehicleRentalSystem
```

Comment:
We need to discuss in the developer meeting what to do with the naming of this field/type and if it's already too late to create a type for this.